scripts/jenkins/Jenkinsfile

```diff
@@ -152,6 +152,10 @@ maybeStage("Deploy Homepage", isMaster()) {
   deployHomepage()
 }
 
+maybeStage("Deploy Web UI", isMaster()) {
```

Comment:
@ingwinlu Is it possible/useful to deploy Homepage/Web UI Demo in parallel?

Reply:
not really as changes to the docker environment are serialised anyway. plus the maximum speed up that can be gained are around 5s.

Reply:
@omnidan I mean this "isMaster" here.